aad/aad-b2c-with-spring-security/aad-b2c-web-application-1-simple-sample/src/main/java/com/spring/sample/b2c/config/WebClientConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.spring.sample.b2c.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.security.oauth2.client.web.OAuth2AuthorizedClientRepository;
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction;
+import org.springframework.web.reactive.function.client.WebClient;
+
+/**
+ * config.
+ */
+@Configuration
+public class WebClientConfig {
```

Comment:
This content is not written in the parent's README.adoc, how about delete these contents in `aad-b2c-web-application-1-simple-sample`, and add these contents in other samples, and add contents in README.adoc to tell customer about these contents.